site/plugins.json

```diff
@@ -653,5 +653,13 @@
     "package": "@vgs/netlify-plugin-vgs",
     "repo": "https://github.com/verygoodsecurity/netlify-plugin-vgs",
     "version": "0.0.2"
+  },
+  {
+    "author": "Rob Grant",
```

Comment:
Is it best to put your name here rather than `TakeShape inc.` or something similar? Out of curiosity